frontend/src/components/TaskMapTask.tsx

```diff
@@ -140,8 +157,18 @@ const SingleTask: FC<
 export const DraggableSingleTask: FC<
   TaskProps & {
     index: number;
+    draggingSomething: boolean;
   }
-> = ({ taskId, index, disableDragging, ...rest }) => (
+> = ({
+  taskId,
+  index,
+  disableDragging,
+  dropDisabled,
+  setGroupDraggable,
+  draggingSomething,
+  isLoading,
+  ...rest
```

Comment:
```suggestion
  ...rest
```

These could be included in the rest, as these are only forwarded